README.mdx

```diff
@@ -1,15 +1,14 @@
 # Mintlify Starter Kit
 
-Click on `Use this template` to copy the Mintlify starter kit. The starter kit
-contains examples including
+Click on `Use this template` to copy the Mintlify starter kit. The starter kit contains examples including
 
 - Guide pages
 - Navigation
 - Customizations
 - API Reference pages
 - Use of popular components
 
-### 👩‍💻 Development
+### ð©âð» Development
```

Comment:
🤔

Reply:
Thanks for pointing this one out, it seems like openai api doesn't understand emojis. I will work on it